src/frontend/contexts/IntlContext.tsx

```diff
@@ -30,7 +30,6 @@ export const IntlProvider = ({children}: {children: React.ReactNode}) => {
 
   return (
     <IntlProviderOrig
-      key={appLocale}
```

Comment:
this was causing essentially the whole app to remount when switching languages. a fun one to debug and eventually find 😄